src/com/yetanalytics/lrs_admin_ui/language.cljs

```diff
@@ -0,0 +1,188 @@
+   :accounts.delete.confirm {:en-US "Are you sure?"}
+   :accounts.password.update {:en-US "Update Password"}
+   :accounts.new.subtitle {:en-US "Create New Account"}
+   :accounts.new.username {:en-US "Username:"}
+   :accounts.new.username.note {:en-US "Username must be %d or more alphanumeric characters"}
+   :accounts.new.password {:en-US "Password:"}
+   :accounts.new.password.note {:en-US "Password must be %d or more characters and contain uppercase, lowercase, numbers, and special characters (%s). Be sure to note or copy the new password as it will not be accessible after creation."}
+   :accounts.new.password.hide {:en-US "Hide"}
+   :accounts.new.password.show {:en-US "Show"}
+   :accounts.new.password.copy {:en-US "Copy"}
+   :accounts.new.password.generate {:en-US "Generate Password"}
+   :accounts.new {:en-US "CREATE ACCOUNT"}
+   ;;Browser
+   :browser.title {:en-US "DATA BROWSER"}
+   :browser.credentials {:en-US "Credentials to Use:"}
+   :browser.query {:en-US "Current Query:"}
+   :browser.key-note {:en-US "Please Choose an API Key Above to Browse LRS Data"}
+   ;;Monitor
+   :monitor.title {:en-US "LRS Monitor"}
+   :monitor.no-data {:en-US "No Statement Data"}
+   :monitor.refresh {:en-US "REFRESH"}
+
+   :monitor.statements.title {:en-US "STATEMENTS"}
+   :monitor.actors.title {:en-US "ACTORS"}
+   :monitor.last-statement.title {:en-us "LAST STATEMENT AT"}
+   :monitor.timeline.title {:en-US "TIMELINE"}
+   :monitor.timeline.unit {:en-US "Time Unit"}
+   :monitor.timeline.since {:en-US "Since"}
+   :monitor.timeline.until {:en-US "Until"}
+
+   :monitor.platform.title {:en-US "PLATFORM"}
+   :monitor.platform.tooltip {:en-US "This metric requires proper use of the “context.platform” field in associated xAPI Statements. If you do not see your connected system represented here, it is possible that it is posting statements that are not using this field."}
+
+   ;;Data Management
+   :datamgmt.title {:en-US "DATA MANAGEMENT"}
+   :datamgmt.delete.title {:en-US "Delete Actor"}
+   :datamgmt.delete.button {:en-US "DELETE"}
+
+   ;;Reactions
+   :reactions.title {:en-us "REACTIONS"}
+   :reactions.title.beta {:en-US " (Beta)"}
+   :reactions.add {:en-US "ADD NEW REACTION"}
+   :reactions.col.title {:en-US "Title"}
+   :reactions.col.conds {:en-US "# of Conditions"}
+   :reactions.col.created {:en-US "Created"}
+   :reactions.col.modified {:en-US "Modified"}
+   :reactions.col.status {:en-US "Status"}
+   :reactions.col.error {:en-US "Error"}
+   :reactions.col.action {:en-US "Action"}
+   :reactions.action.edit {:en-US "Edit"}
+   :reactions.action.delete {:en-US "Delete"}
+
+   :reactions.new.title {:en-US "New Reaction"}
+   :reactions.edit.title {:en-US "Edit Reaction"}
+   :reactions.focus.title {:en-US "Reaction Details"}
+
+   :reactions.details.created {:en-US "Created"}
+   :reactions.details.modified {:en-US "Modified"}
+   :reactions.details.error {:en-US "Error"}
+   :reactions.details.title {:en-US "Error"}
```

Comment:
Wait sorry I overlooked this during my review - shouldn't the second line say "Title"?

Reply:
yup, good catch, 525b608b49f3f38609a3b6e3b303661972339d63

Reply:
The error was indeed an error. The irony.